docs/spec.md

```diff
@@ -1303,20 +1305,21 @@ implementation-defined as well.
 
 #### Inputs
 
-| Label | Name      | Type   | Constraints |
-|-------|-----------|--------|-------------|
-| (I1)  | `operand` | tensor | (C1), (C2)  |
+| Label | Name      | Type                                  | Constraints |
+|-------|-----------|---------------------------------------|-------------|
+| (I1)  | `operand` | tensor or per-tensor quantized tensor | (C1-C2)     |
```

Comment:
Since this is a reinterpret cast, let's support all quantized tensors?

Reply:
agree. done.